src/main/resources/META-INF/plugin.xml

```diff
@@ -14,24 +14,17 @@
 
     <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
          on how to target different products -->
-    <idea-version since-build="131"/>
+    <idea-version since-build="201.5985.32"/>
```

Comment:
Users with pre 2020.1 should start screaming right now 😄

Can I merge and release #4 first, before yours?

Reply:
As you wish, but I think it's now ok to drop 2019.3 support (we're already dropped in Rust plugin)